src/Internal.AspNetCore.Sdk/build/Common.targets

```diff
@@ -56,4 +58,13 @@ For single-tfm projects, this will be imported from build/Internal.AspNetCore.Sd
     </Sdk_GenerateResxDesignerFiles>
   </Target>
 
+  <Target Name="_AppendRepositoryCommitUrlToPackageDescription"
+    DependsOnTargets="ResolveCommitHash"
+    Condition="'$(GenerateRepositoryCommitUrlInDescription)' == 'true'">
+    <PropertyGroup>
+      <PackageDescription>$(PackageDescription)
+Repository commit url: $(RepositoryUrl)/tree/$(CommitHash)</PackageDescription>
```

Comment:
Nit: can you use `$(RepositoryCommit)` instead?

Reply:
`$(CommitHash)` was the old property name which is only present for compatibility. I was hoping to remove it as a part of cleanup in 2.2.